src/essentia/streaming/streamingalgorithm.cpp

```diff
@@ -33,7 +33,9 @@ void connect(SourceBase& source, SinkBase& sink) {
     //     this is a requirement for ProxyConnectors
     E_DEBUG(EConnectors, "Connecting " << source.fullName() << " to " << sink.fullName());
     sink.connect(source);
+    E_DEBUG(EConnectors, "  " << source.fullName() << " ok");
     source.connect(sink);
+    E_DEBUG(EConnectors, "  " << sink.fullName() << " ok");
```

Comment:
Are these new `E_DEBUG`s necessary? They clutter the output.

Reply:
No idea what this is. Removed in 63634927f0b3cf1bfd14ae7184ffc770e940ff17